journals/journal_abbreviations_annee-philologique.txt

```diff
@@ -830,8 +836,3 @@ Zentralblatt für Bibliothekswesen = ZBB
 Ziva Antika (Antiquité vivante) = ZAnt
 Zprávy Jednoty Klasickaych Filologu = ZJKF
 Échos du monde classique (Classical Views) = EMC
```

Comment:
This entry should also be moved up to adhere to the alphabetical order, shouldn't it? 

Could you please do that as well, then I will merge your PR.